vaadin-form-layout-flow-parent/vaadin-form-layout-flow-integration-tests/src/main/java/com/vaadin/flow/component/formlayout/test/FormLayoutView.java

```diff
@@ -155,13 +153,10 @@ public void addItemWithLabel(String label, Component... items) {
             getContent().addFormItem(itemWrapper, label);
         }
     }
-    // end-source-example
 
     private void createResponsiveLayout() {
         // @formatter:off
-        // begin-source-example
-        // source-example-heading: A form layout with custom responsive layouting
-        FormLayout nameLayout = new FormLayout();
+                        FormLayout nameLayout = new FormLayout();
```

Comment:
This line looks wrong, maybe remove `@formatter:off` above and then format again.

Reply:
Done